Lib/fontmake/font_project.py

```diff
@@ -577,6 +589,7 @@ def save_otfs(
                     ufo, ext, is_instance, autohinted=True, output_dir=output_dir
                 )
             try:
+                logger.info("Autohinting %s", otf_path)
                 ttfautohint(otf_path, hinted_otf_path, args=autohint)
```

Comment:
autohint_thisfont

Reply:
that is why one needs tests 👍

Reply:
Please be specific. I don't understand the issue here

Reply:
Oh, I get it. Simon marked the wrong line. the ttfautohint() needs the new variable instead.

Reply:
sorry. You are not using the `autohint_thisfont` local variable that you just made exactly where you need it most: i.e. to the args parameter of ttfautohint. It is still using the global `autohint` one